packages/manager/apps/pci-public-ip/src/interface/index.ts

```diff
@@ -19,3 +19,32 @@ export interface FloatingIP {
   region: string;
   status: string;
 }
+
+export interface FailoverIP {
+  associatedEntity: Instance | null;
+  block: string;
+  continentCode: 'EU';
+  geoloc: 'FR';
```

Comment:
type string ?

Reply:
My bad, it was a copy/paste from network 🥇